tests/test_grid.py

```diff
@@ -459,19 +465,26 @@ def test_grid_points():
                        [1, 1, 1],
                        [0, 1, 1]])
     grid = pyvista.UniformGrid()
-    grid.points = points
-    assert grid.dimensions == [2, 2, 2]
-    assert grid.spacing == [1, 1, 1]
-    assert grid.origin == [0., 0., 0.]
+    grid.dimensions = [2, 2, 2]
+    grid.spacing = [1, 1, 1]
+    grid.origin = [0., 0., 0.]
     assert np.allclose(np.unique(grid.points, axis=0), np.unique(points, axis=0))
     opts = np.c_[grid.x, grid.y, grid.z]
     assert np.allclose(np.unique(opts, axis=0), np.unique(points, axis=0))
+
     # Now test rectilinear grid
     del grid
```

Comment:
Same here.  We might consider having separate tests as unit tests are generally more isolated in scope.